pkg/plugins/resources/postgres/migrations/data/1589041445_add_unique_id_and_owner.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE resources
+    DROP CONSTRAINT resources_pkey,
+    ADD CONSTRAINT resources_pkey PRIMARY KEY (name, mesh, type);
+ALTER TABLE resources DROP COLUMN namespace;
+ALTER TABLE resources ADD COLUMN owner_name varchar(100);
+ALTER TABLE resources ADD COLUMN owner_mesh varchar(100);
+ALTER TABLE resources ADD COLUMN owner_type varchar(100);
+ALTER TABLE resources ADD CONSTRAINT owner_fk FOREIGN KEY (owner_name,owner_mesh,owner_type) REFERENCES resources(name,mesh,type) ON DELETE CASCADE;
```

Comment:
Yes, owner is optional since Mesh doesn't have it. I updated the migration script that does what we discussed offline:
1. Delete DataplaneInsights
2. Tries to set owners
3. Delete dangling resources (without owner)